README.md

```diff
@@ -27,6 +30,37 @@ gitmoji-changelog
 
 **// TODO**
 
+## Setup
+
+Install the lib in your PATH : 
+
+```bash
+npm install -g gitmoji-changelog
+```
+
+## Mapping
+
+When a commit is done with a gitmoji, the following mapping is used to group them : 
+
+| Categorie | Gitmoji |
+|---|---|
+| Added | :sparkles: :tada: :white_check_mark: :bookmark: :construction_worker: :charts_with_upwards_trend: :heavy_plus_sign: :loud_sound: |
```

Comment:
Small typo here `chart_with_upwards_trend` 😄

Reply:
Ok, so there is a typo in `mapping.js`, I'll make another PR with this correction